test/signs_ui/config/expiration_test.exs

```diff
@@ -50,9 +51,141 @@ defmodule SignsUi.Config.ExpirationTest do
+            "5678" => %{}
+          },
+          "Blue" => %{
+            "abc" => %{}
+          }
+        }
+      }
+
+      alert_ids = SignsUi.Config.Expiration.active_alert_ids(alert_state)
+      assert alert_ids == MapSet.new(["1234", "5678", "abc"])
+
+      sign_state = %{
+        signs: %{
+          "250" => %Sign{
+            config: %{mode: :auto},
+            id: "250"
+          },
+          "460" => %Sign{
+            config: %{
+              alert_id: nil,
+              expires: nil,
+              mode: :headway
+            },
+            id: "460"
+          },
+          "898" => %Sign{
+            config: %{
+              alert_id: "1234",
+              expires: nil,
+              mode: :headway
+            },
+            id: "898"
+          },
+          "925" => %Sign{
+            config: %{
+              alert_id: "326",
+              expires: nil,
+              mode: :headway
+            },
+            id: "925"
+          },
+          "793" => %Sign{
+            id: "793",
+            config: %{
+              mode: :off,
+              expires: nil,
+              alert_id: nil
+            }
+          },
+          "367" => %Sign{
+            id: "367",
+            config: %{
+              mode: :off,
+              expires: nil,
+              alert_id: "5678"
+            }
+          },
+          "471" => %Sign{
+            id: "471",
+            config: %{
+              mode: :off,
+              expires: nil,
+              alert_id: "437"
+            }
+          },
+          "714" => %Sign{
+            id: "714",
+            config: %{
+              mode: :static_text,
+              line1: "test",
+              line2: "test",
+              expires: nil,
+              alert_id: nil
+            }
+          },
+          "474" => %Sign{
+            id: "474",
+            config: %{
+              mode: :static_text,
+              line1: "test",
+              line2: "test",
+              expires: nil,
+              alert_id: "abc"
+            }
+          },
+          "273" => %Sign{
+            id: "273",
+            config: %{
+              mode: :static_text,
+              line1: "test",
+              line2: "test",
+              expires: nil,
+              alert_id: "593"
+            }
+          }
+        }
+      }
+
+      expired_signs =
+        SignsUi.Config.Expiration.expire_signs(
+          sign_state,
+          fn -> DateTime.new!(~D[2019-01-15], ~T[08:00:00], "America/New_York") end,
+          fn -> MapSet.new(["1234", "5678", "abc"]) end
+        )
+
+      expected_signs = %{
+        "925" => %Sign{
+          id: "925",
+          config: %{mode: :auto}
+        },
+        "471" => %Sign{
+          id: "471",
+          config: %{mode: :auto}
+        },
+        "273" => %Sign{
+          id: "273",
+          config: %{mode: :auto}
+        }
+      }
+
+      assert MapSet.new(expired_signs) == MapSet.new(expected_signs)
```

Comment:
No need to do `MapSet` here. We only did that for the other tests because the return value of the function actually was a MapSet. But here they're not. You should just be able to do `assert expired_signs == expected_signs`.